labelbox/data/serialization/ndjson/base.py

```diff
@@ -1,3 +1,4 @@
+from typing import Optional
```

Comment:
writing the comment here, but it looks like the test cases are failing as a result of these changes. they will likely need some updates